### PR TITLE
Added a 1 quote

### PR DESCRIPTION
**Pull request by chatbox9321**
Added a quote, one of the best lines ever written in movies.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
 <title>Practise for git</title>
 </head>
 <body>
-<h1>Hello.</h1>
+<h1>The way this works is you do the thing that you are scared shitless of and you get the courage after you do it not before. Geroge Clooney (Three kings)</h1>
```
Comment (Avry123) [inline index.html:6]: Don't need the h1 tag here.

**Review by Avry123 — COMMENTED** [review]
Please neel15 there is a change.

File: index.html (hunk 1 above)
Comment (chatbox9321) [reply index.html:6]: Alright, I will take care of it.

**Review by Avry123 — APPROVED** [review]
Well done.

**Pull request merged**